src/libstd/io/util.rs

```diff
@@ -78,14 +78,15 @@ pub struct Empty { _priv: () }
 /// A slightly sad example of not reading anything into a buffer:
 ///
 /// ```
-/// use std::io;
-/// use std::io::Read;
+/// use std::io::{self, Read};
 ///
 /// # fn foo() -> io::Result<String> {
 /// let mut buffer = String::new();
 /// try!(io::empty().read_to_string(&mut buffer));
```

Comment:
Should all these `try!`s be converted to unwraps? That seems to be the modern convention?